pom.xml

```diff
@@ -19,7 +19,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <abstract-operator.version>0.6.14</abstract-operator.version>
+        <abstract-operator.version>0.6.16</abstract-operator.version>
```

Comment:
will change this to whatever the right version is after the abstract operator PR is merged